### DISCUSSION TITLE
Why calling getDisplayMedia() cannot pop up a screen sharing dialog box, and how to achieve screen sharing

### DISCUSSION THREAD

**Discussion by ideniden** [cefsharp/CefSharp#4984] [General]
please,thanks...

**Comment by amaitland** [comment]
Are you using `CefSharp`?

**Comment by ideniden** [comment]
yes,I am.

**Comment by amaitland** [comment]
It depends if you are using `WinForms` or `WPF`.

There is no default dialog (and as a result is disabled by default).

You can implement [PermissionHandler](https://cefsharp.github.io/api/130.1.x/html/T_CefSharp_Handler_PermissionHandler.htm) and assign to the browser like the folowing:

```
chromiumWebBrowser.PermissionHandler = new MyPermissionHandler();
```

If you are using `WinForms` then you can probably switch to the `Chrome styled` windows, should be able to use the built in dialog. (This is new and not something I've tried yet).

**Reply by ideniden** [reply]
Thank you so much, my problem has been solved!
